docs/development/help-wanted.md

```diff
@@ -0,0 +1,7 @@
+# Help Wanted!
+
+This project,  [mattermost-plugin-gitlab](https://github.com/mattermost/mattermost-plugin-gitlab),  is a fork of the [mattermost-plugin-github](https://github.com/mattermost/mattermost-plugin-github). Thanks to all contributors of it.
+
+If you're interested in joining our community of developers who contribute to Mattermost - check out the current set of issues [that are being requested](https://github.com/mattermost/mattermost-plugin-gitlab/issues?q=is%3Aissue+is%3Aopen+label%3AEnhancement).
+
+You can also find issues labeled ["Help Wanted"](https://github.com/mattermost/mattermost-plugin-gitlab/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22) in the Jira Repository that we have laid out the primary requirements for and could use some coding help from the community.
```

Comment:
```suggestion
You can also find issues labeled ["Help Wanted"](https://github.com/mattermost/mattermost-plugin-gitlab/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22) in the GitHub repository that we have laid out the primary requirements for and could use some coding help from the community.
```

Reply:
Ah, I just used the one from the Jira plug-in and forgot to change the Jira reference in that case to GitLab.  Will change it to GitLab plugin repository.